### ISSUE TITLE
Add support for class-level docstring

### ISSUE THREAD

**Issue by edmondchuc** [simon-ritchie/stubdoc#5]
Hi! Awesome library.

Currently, stubdoc only supports adding docstring to functions and class methods.

https://github.com/simon-ritchie/stubdoc/blob/0e77e00e537e71482461b36b55983fd6e90c99e7/stubdoc/stubdoc.py#L37-L48

It would be great if it supported class-level docstrings as well as class getters and setters (`@property`).

Stub file with docstring generated by stubdoc:

```python
class NamedNode:
    @property
    def value(self) -> str: ...
    def __init__(self, value: str) -> None:
        """
        Initialize self.  See help(type(self)) for accurate signature.
        """
    def __str__(self) -> str: ...
    def __repr__(self) -> str: ...
    def __hash__(self) -> int: ...
    def __eq__(self, other: Any) -> bool: ...
    def __ge__(self, other: Any) -> bool: ...
    def __gt__(self, other: Any) -> bool: ...
    def __le__(self, other: Any) -> bool: ...
    def __lt__(self, other: Any) -> bool: ...
    def __ne__(self, other: Any) -> bool: ...
```

Desired output:

```python
class NamedNode:
    """An RDF `node identified by an IRI <https://www.w3.org/TR/rdf11-concepts/#dfn-iri>`_.

    :param value: the IRI as a string.
    :raises ValueError: if the IRI is not valid according to `RFC 3987 <https://tools.ietf.org/rfc/rfc3987>`_.

    The :py:func:`str` function provides a serialization compatible with N-Triples, Turtle, and SPARQL:

    >>> str(NamedNode('http://example.com'))
    '<http://example.com>'
    """
    @property
    def value(self) -> str:
        """the named node IRI.
        
        >>> NamedNode("http://example.com").value
        'http://example.com'
        """
    def __init__(self, value: str) -> None:
        """
        Initialize self.  See help(type(self)) for accurate signature.
        """
    def __str__(self) -> str: ...
    def __repr__(self) -> str: ...
    def __hash__(self) -> int: ...
    def __eq__(self, other: Any) -> bool: ...
    def __ge__(self, other: Any) -> bool: ...
    def __gt__(self, other: Any) -> bool: ...
    def __le__(self, other: Any) -> bool: ...
    def __lt__(self, other: Any) -> bool: ...
    def __ne__(self, other: Any) -> bool: ...
```

**Comment by simon-ritchie** [comment]
Hi @edmondchuc 

Thanks for posting the nice proposal

Currently, I'm too busy and probably it takes a long time to support it  
Please be patient to support its feature

P.S. If you favorite this project, please leave a star⭐️  
It motivate me to support this proposal

**Comment by simon-ritchie** [comment]
I've released the v0.1.11 and probably supported these features!

https://github.com/simon-ritchie/stubdoc/releases/tag/0.1.11

```
$ pip install stubdoc==0.1.11
```

If you notice this version does not work correctly, please notify me
